packages/dapp-toolkit/src/wallet-request/request-items/request-item-client.ts

```diff
@@ -107,21 +99,21 @@ export const RequestItemClient = (input: RequestItemClientInput) => {
       })
   }
 
-  const getPendingItems = () =>
+  const getPendingRequests = () =>
```

Comment:
Do we distinguish between requests/transactions in any way? Maybe we should stick to using `interactions` if we mean either one or another?

Reply:
Request item and wallet interaction is different

Reply:
<img width="325" alt="image" src="https://github.com/radixdlt/radix-dapp-toolkit/assets/5996605/a3e46142-c0df-4cba-8d5b-abcd71a1a956">
<img width="532" alt="image" src="https://github.com/radixdlt/radix-dapp-toolkit/assets/5996605/75a81c09-f89a-4fc2-bc60-e87b985699c1">